nncf/experimental/openvino_native/quantization/default_quantization.py

```diff
@@ -58,6 +58,8 @@
         ov_metatypes.OVExpMetatype,
         ov_metatypes.OVLogMetatype,
         ov_metatypes.OVAssignMetatype,
+        ov_metatypes.OVReluMetatype,
+        ov_metatypes.OVDeformableConvolutionMetatype,
```

Comment:
Please, add the same changes to ONNX, Torch and TensorFlow backends.

Reply:
Added